### PR TITLE
Add cf-cli-eng bot to App Runtime Interfaces

### PR DESCRIPTION
**Pull request by a-b**
cf-cli-eng is used in CF CLI automations

cc @moleske

### CODE REVIEW

**Comment by a-b** [comment]
@stephanme, could you please review this pull request since our working group's tech lead is on leave?

**Comment by beyhan** [comment]
@moleske could you please put a thumbs up or approval on this. 

@stephanme could review or I will add this to be reviewed by the TOC.

File: toc/working-groups/app-runtime-interfaces.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ bots:
   github: capi-bot
 - name: ARI WG Git Bot
   github: ari-wg-gitbot
+- name: CF CLI Eng
+  github: cf-cli-eng
```
Comment (stephanme) [inline toc/working-groups/app-runtime-interfaces.md:53]: Is there a real need to add cf-cli-eng as WG bot and not as WG area bot just for the CLI area?
See Notification WG area for an example.

There are more bot users that should be moved from WG into WG area. I will put this topic on the agenda for the next WG meeting.

**Review by stephanme — CHANGES_REQUESTED** [review]

File: toc/working-groups/app-runtime-interfaces.md (hunk 1 above)
Comment (moleske) [reply toc/working-groups/app-runtime-interfaces.md:53]: I didn't realize you could scope bots to more specific areas.  Here are the example lines https://github.com/cloudfoundry/community/blob/522836cdbcba75494e25ebfce10e2ef42777ae46/toc/working-groups/app-runtime-interfaces.md?plain=1#L432-L435

**Review by moleske — APPROVED** [review]
This seems reasonable, given we make the change @stephanme is making about scoping it to the cli area rather than all of app runtime interfaces

**Comment by a-b** [comment]
Addressed @stephanme suggestion and created follow up #781 for CAPI

**Pull request merged**